Annex60/Fluid/Actuators/BaseClasses/PartialDamperExponential.mo

```diff
@@ -3,18 +3,14 @@ partial model PartialDamperExponential
   "Partial model for air dampers with exponential opening characteristics"
    extends Annex60.Fluid.BaseClasses.PartialResistance(
       m_flow_turbulent=if use_deltaM then deltaM * m_flow_nominal else
-      eta_default*ReC*sqrt(area)*facRouDuc);
+      eta_default*ReC*sqrt(A)*facRouDuc);
    extends Annex60.Fluid.Actuators.BaseClasses.ActuatorSignal;
  parameter Boolean use_deltaM = true
     "Set to true to use deltaM for turbulent transition, else ReC is used";
  parameter Real deltaM = 0.3
     "Fraction of nominal mass flow rate where transition to turbulent occurs"
    annotation(Dialog(enable=use_deltaM));
- parameter Boolean use_v_nominal = true
-    "Set to true to use face velocity to compute area";
- parameter Modelica.SIunits.Velocity v_nominal=1 "Nominal face velocity"
```

Comment:
I suggest to leave this parameter, and use it below in `A=m_flow_nominal/rho_default/v_nominal`